test/e2e/portforward.go

```diff
@@ -77,7 +81,52 @@ func pfPod(expectedClientData, chunks, chunkSize, chunkIntervalMillis string) *a
 	}
 }
 
-func runPortForward(ns, podName string, port int) (*exec.Cmd, int) {
+type portForwardContext struct {
+	cmd    *exec.Cmd
+	port   int
+	stdout io.Closer
+	stderr io.Closer
```

Comment:
Assigned back to Cmd.Stdout though: https://golang.org/src/os/exec/exec.go?s=13981:13995#L497